contracts/0.8.25/vaults/LazyOracle.sol

```diff
@@ -379,6 +396,9 @@ contract LazyOracle is ILazyOracle, AccessControlEnumerableUpgradeable {
         if (_cumulativeLidoFees - previousCumulativeLidoFees > maxLidoFees) {
             revert CumulativeLidoFeesTooLarge(_cumulativeLidoFees - previousCumulativeLidoFees, maxLidoFees);
         }
+
+        // 5. _maxLiabilityShares is greater than _liabilityShares
+        if (_maxLiabilityShares < _liabilityShares) revert InvalidMaxLiabilityShares();
     }
```

Comment:
Shall we add a sanity check that report's record.maxLiabilityShares can't be LESS than the report.maxLiabilityShares?

Reply:
such a report wouldn't have been dangerous except a logic invariant violation